app/src/main/java/nu/yona/app/api/service/ActivityMonitorService.java

```diff
@@ -106,15 +114,50 @@ private void restartReceiver()
 	public int onStartCommand(Intent intent, int flags, int startId)
 	{
 		scheduleMethod();
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+		{
+			displayActivityMonitoringNotification();
+		}
 		return START_NOT_STICKY;
 	}
 
+	@TargetApi(26)
+	private void displayActivityMonitoringNotification()
+	{
+		String CHANNEL_ID = "yona-channel";
+		NotificationChannel channel = new NotificationChannel(CHANNEL_ID,
+				"yona activity monitoring channel",
```

Comment:
According to [the documentation](https://developer.android.com/reference/android/app/NotificationChannel#NotificationChannel(java.lang.String,%20java.lang.CharSequence,%20int)), this name is user-visible, so please make it localizable.

Reply:
Do you want `yona-channel` also to be translated. google gives me `yona-kanaals`. Is that OK.

Reply:
The ID is not user-visible and thus doesn't need to be translatable.

Reply:
Sorry. wrong question. I did not read it properly.